apps/server/src/modules/files-storage-client/interfaces/file-domain-object-props.ts

```diff
@@ -1,10 +1,8 @@
-import { EntityId } from '@shared/domain';
-import { FileRecordParamsParentTypeEnum } from '../filesStorageApi/v3';
+import { EntityId, FileRecordParentType } from '@shared/domain';
 
 export interface IFileDomainObjectProps {
```

Comment:
This interface is named with a leading "I" unlike e.g. FileRequestInfo. Is there a special reason for it here? Do we have a standard for that?